figures/scripts/fig1-sample-distribution.R

```diff
@@ -0,0 +1,142 @@
+# Sample Distribution Figure
+#
+# 2020
+# Chante Bethell for ALSF - CCDL
+#
+# This script is intended to run steps needed to create Figure 1.
+
+# Load in libraries
+library(dplyr)
+library(colorspace)
+library(scales)
+library(treemapify)
+
+# Magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+# Detect the ".git" folder -- this will in the project root directory.
+# Use this as the root directory to ensure proper execution, no matter where
+# it is called from.
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# Declare output directory
+output_dir <- file.path(root_dir, "figures", "pngs")
+
+#### Declare relative file paths of modules used for this figure --------------
+sample_distribution_dir <- file.path(
+  root_dir,
+  "analyses",
+  "sample-distribution-analysis"
+)
+
+## TODO: Define the file path to the directory containing the data for the
+##        contributions plot
+
+#### Read in the associated results -------------------------------------------
+
+# Read in disease expression file from `01-filter-across-types.R`
+disease_expression <-
+  readr::read_tsv(file.path(sample_distribution_dir, "results", "disease_expression.tsv"))
+
+# Read in plots data.frame file from `02-multilaye-plots.R`
+plots_df <-
+  readr::read_tsv(file.path(sample_distribution_dir, "results", "plots_df.tsv"))
+
+# Reorder the columns to be displayed in descending order by count on the plot
+disease_expression$integrated_diagnosis <- with(disease_expression,
+                                                reorder(integrated_diagnosis, -count))
+
+#### Re-run the individual plots ----------------------------------------------
+
+# Create a bar plot of sample distribution across cancer types
+gg_types_bar <- disease_expression %>%
+  ggplot2::ggplot(ggplot2::aes(x = integrated_diagnosis, y = count, fill = count)) +
```

Comment:
I don't tend to like having redundant info in a figure, so having the fill also represent the count is probably not needed here.  However, we may well want to use the colors from the final color schemes https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/510 to color the bars by histology.  Either way, we probably don't want to have the color legend at all, as it does not provide any real information, so you will want to add to theme `legend.position = 'none'`